src/server/ApiRoutes/Bib.js

```diff
@@ -116,6 +116,9 @@ const addLocationUrls = (bib) => {
 };
 
 function fetchBib (bibId, cb, errorcb, reqOptions, res) {
+  // Redirect if bibId has a Check Digit
+  if (hasCheckDigit(bibId)) { res.redirect(`${appConfig.baseUrl}/bib/${removeCheckDigit(bibId)}`); }
```

Comment:
NBD but `removeCheckDigit` internally also checks for `hasCheckDigit`. Don't think you can get around it for this case.